docs/architecture.md

```diff
@@ -32,3 +32,11 @@ related:
 * {{ "lobby" | dfn }}
 * {{ "reception" | dfn }}
 * {{ "room" | dfn }}
+
+## Stages
```

Comment:
If we make this a level 3 heading, and move these new terms below ‘Construction’, I think that would work better.